jmsSource/build.gradle

```diff
@@ -0,0 +1,96 @@
+group 'io.vantiq'
+version 'unspecified'
+
+apply plugin: 'java'
+apply plugin: 'application'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+    jcenter()
+
+    // Used for checking VANTIQ date format behavior with VANTIQ SDK
+    maven {
+        url "https://dl.bintray.com/vantiq/maven"
+    }
+}
+
+mainClassName = 'io.vantiq.extsrc.jmsSource.JMSMain'
+
+startScripts{
+    doLast{
+        def windowsScriptFile = file getWindowsScript()
+        def unixScriptFile = file getUnixScript()
+        // Add the log Config to the top of the classpath
+        windowsScriptFile.text = windowsScriptFile.text.replace("CLASSPATH=", "CLASSPATH=%APP_HOME%\\logConfig;")
+        unixScriptFile.text = unixScriptFile.text.replace('CLASSPATH=', 'CLASSPATH=$APP_HOME/logConfig:')
+    }
+}
+
+// Copies the logger setup into logConfig in the distribution
+applicationDistribution.from("src/main/resources") {
+    include "log4j2.xml"
+    into "logConfig"
+}
+
+tasks.withType(Test) {
+    if (System.env.JMS_DRIVER_LOC == null) {
```

Comment:
do we have a mechanism to run tests against a variety of these?  Some über script that rotates through our test subjects?  May require some machine with them all set up?